apps/www/components/Climatelab/Postcard/PostcardGenerator.js

```diff
@@ -47,9 +47,32 @@ const SubmittedPostcard = (props) => {
     scrollIntoView(postcardRef.current)
   }, [])
 
+  console.log(questionnaire)
```

Comment:
a wild log appeared…

Reply:
shall be removed, i think i forgot it for some weird reason